packages/wmr/src/wmr-middleware.js

```diff
@@ -67,6 +67,7 @@ export default function wmrMiddleware({
 
 	const NonRollup = createPluginContainer(
 		[
+			...plugins.filter(p => p.enforce === 'pre'),
```

Comment:
Idea: I believe `plugins` is already sorted by `enforce` here, so we could do:

```js
const split = plugins.findIndex(p => p.enforce !== 'pre');

const NonRollup = createPluginContainer(
	[
		...plugins.slice(0, split), // "pre"
		// ... etc
		...plugins.slice(split)  // undefined/"normal", then "post"
	],
	// ... etc
);

```

Reply:
That won't work without reassigning the `plugins` variable after each slice operation.